apps/dashboard/lib/trpc/ratelimitProcedure.ts

```diff
@@ -0,0 +1,57 @@
+// Values for route types
+import { protectedProcedure } from "./trpc";
+import { Ratelimit } from "@unkey/ratelimit";
+import { env } from "../env";
+import { TRPCError } from "@trpc/server";
+
+//Values are in seconds currently
```

Comment:
do we need them defined here?
why don't we just use the values directly in the `new Ratelimit` calls?